nixos/tests/common/letsencrypt/default.nix

```diff
@@ -135,47 +135,15 @@ let
     ];
   };
 
-  boulder = let
-    owner = "letsencrypt";
-    repo = "boulder";
-    rev = "9c6a1f2adc4c26d925588f5ae366cfd4efb7813a";
-    version = "20180129";
-
-  in pkgs.buildGoPackage rec {
-    name = "${repo}-${version}";
-
-    src = pkgs.fetchFromGitHub {
-      name = "${name}-src";
-      inherit rev owner repo;
-      sha256 = "09kszswrifm9rc6idfaq0p1mz5w21as2qbc8gd5pphrq9cf9pn55";
-    };
-
+  boulder = pkgs.boulder.overrideAttrs (superAttrs:
+  let goPackagePath = "github.com/letsencrypt/boulder";
+  in {
+    inherit goPackagePath;
     postPatch = ''
-      # compat for go < 1.8
-      sed -i -e 's/time\.Until(\([^)]\+\))/\1.Sub(time.Now())/' \
-        test/ocsp/helper/helper.go
-
-      find test -type f -exec sed -i -e '/libpkcs11-proxy.so/ {
-        s,/usr/local,${pkcs11-proxy},
-      }' {} +
-
-      sed -i -r \
-        -e '/^def +install/a \    return True' \
-        -e 's,exec \./bin/,,' \
-        test/startservers.py
-
       cat ${lib.escapeShellArg snakeOilCerts.ca.key} > test/test-ca.key
       cat ${lib.escapeShellArg snakeOilCerts.ca.cert} > test/test-ca.pem
     '';
 
-    # Until vendored pkcs11 is go 1.9 compatible
-    preBuild = ''
-      rm -r go/src/github.com/letsencrypt/boulder/vendor/github.com/miekg/pkcs11
-    '';
-
-    # XXX: Temporarily brought back putting the source code in the output,
```

Comment:
Do not remove the explanation. Or maybe change it to the first paragraph of https://github.com/NixOS/nixpkgs/commit/c21b1ede95de8bead6a83b8f182fd17f2c6a0ee0 to be even clearer.